### ISSUE TITLE
Bug in devtools::run_examples() devtools_2.0.1

### ISSUE THREAD

**Issue by zellerivo** [r-lib/pkgload#107]
run_examples(run= FALSE) does what run_examples(run=TRUE) is actually supposed to do, it comments out code in \dontrun{}.

**Comment by jimhester** [comment]
This is actually a bug in pkgload, these lines should not have ! before them

https://github.com/r-lib/pkgload/blob/78e7e3acf775268ccf58f420e228541fd549877a/R/dev-example.r#L48-L50

**Comment by kwstat** [comment]
I'm confused by the wording of the documentation in regards to the run/test arguments.  I would like to suggest that the wording be changed to be similar to what is shown by Rcmd check:  
```
Rcmd check --help
      --run-dontrun     do run \dontrun sections in the Rd files
      --run-donttest    do run \donttest sections in the Rd files
```

Something like this makes more sense to me:
```
#' @param test if `TRUE`, do run code in \code{\\donttest{}} sections.
#' @param run if `TRUE`, do run code in \code{\\dontrun{}} sections.
```

**Comment by salim-b** [comment]
> Something like this makes more sense to me:
> 
> ```rd
> #' @param test if `TRUE`, do run code in \code{\\donttest{}} sections.
> #' @param run if `TRUE`, do run code in \code{\\dontrun{}} sections.
> ```

Why not:

```rd
#' @param run_donttest if `TRUE`, do run code in \code{\\donttest{}} sections.
#' @param run_dontrun if `TRUE`, do run code in \code{\\dontrun{}} sections.
```
IMHO, the clarity gained would by far outweigh the additional typing required.

**Comment by lionel-** [comment]
Fixed by r-lib/devtools#2146 and https://github.com/r-lib/pkgload/commit/3347012a9fff4318554b58e27dc6fef441c25ed8.
